javascript/data.js

```diff
@@ -12,6 +12,7 @@
 // var portfolioData = [testOne, testTwo, testThree, testFour, testFive];
 
 var rawData = [
+  {"name":"Test","link":"www.google.com","img":"images/template.png","body":"Here is a sample portfolio object I made!","category":"portfolioPiece","date":"3/20/2017"},
```

Comment:
You need single quotes instead of double. I am getting a lot of errors on my linter because of this.